frontend/src/shared/components/Container.tsx

```diff
@@ -1,9 +1,11 @@
 import type { ReactNode } from "react";
 
 type Props = {
-  children: ReactNode
+  children: ReactNode,
+  className?: string
 }
 
-export function Container({ children }: Props) {
-  return <div className="mx-auto max-w-[1020px]">{children}</div>;
+export function Container({ children, className }: Props) {
+  const containerClassName = "max-w-[1020px] mx-auto ";
+  return <div className={containerClassName + (className === undefined ? "" : className)}>{children}</div>;
```

Comment:
プロジェクトに既に`classnames`がインストールされてるから、これを使うと良しなにできるよ！

```suggestion
  return <div className={classNames(containerClassName, className)}>{children}</div>
```

ref: [classnames](https://github.com/JedWatson/classnames)

Reply:
修正しました